components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -1219,3 +1224,8 @@ func runAsGitpodUser(cmd *exec.Cmd) *exec.Cmd {
 	cmd.SysProcAttr.Credential.Gid = gitpodGID
 	return cmd
 }
+
+func handleExit(ec *int) {
+	exitCode := *ec
```

Comment:
I would love to keep the Debug line from here https://github.com/gitpod-io/gitpod/pull/5240/files#diff-38855a295b4f047d2f93ba019f506a7d51edef9c2aed4c47cfcc2f0a01bf3835L288

Reply:
done